src/devices/Card/Ndef/GeneralePurposeByteConditions.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+
+namespace Iot.Device.Ndef
+{
+    /// <summary>
+    /// This is used to asses the block sector and understand the read/write conditions
+    /// </summary>
+    [Flags]
+    public enum GeneralePurposeByteConditions
+    {
+        /// <summary>
+        /// No read access
+        /// </summary>
+        NoReadAccess = 0b0000_1100,
```

Comment:
don't understand this. Isn't lack of ReadAccess flag equivalent to NoReadAccess?

Reply:
(same comment for WriteAccess)

Reply:
Well, it's complicated with Mifare cards. NDEF require specific flags. So here they are. That does allow to check if you can write or read. Not in the samples as a bit advance but if you start to read a lot of cards, want to write if possible, etc, that will be super useful.

Reply:
And I know it may seems strange but they've designed it like that with those access bits.